app/models/wishlist_item.rb

```diff
@@ -12,10 +12,12 @@ class WishlistItem < ApplicationRecord
             format: {
               # allows 1 to 61 characters with Estonian diacritic signs and 1 to 61 character
               # of the top level domain.
-              with: /\A[a-z0-9\-\u00E4\u00F5\u00F6\u00FC\u0161\u017E]{1,61}\.[a-z0-9]{1,61}\z/,
+              with: /\A[a-z0-9\-\u00E4\u00F5\u00F6\u00FC\u0161\u017E]{1,61}\.
```

Comment:
Is it the same regexp as on https://github.com/internetee/auction_center/pull/384/files#diff-8c3f2a547e7f7d66e1756971a1ef20c2R117?

Reply:
No, it's not. The one in `Setting` is limited to one subdomain (ee, pri.ee). `WishlistItem` regex doesn't have such limitation.

Reply:
Same as above: can we extract this regexp somewhere and name it based on its semantic meaning?